static/app/components/events/interfaces/spans/spanSiblingGroupBar.tsx

```diff
@@ -135,7 +141,14 @@ export default function SpanSiblingGroupBar(props: Props) {
       treeDepth={props.treeDepth}
       spanNumber={spanNumber}
       generateBounds={generateBounds}
-      toggleSpanGroup={() => toggleSiblingSpanGroup?.(spanGrouping[0].span, occurrence)}
+      toggleSpanGroup={() => {
+        toggleSiblingSpanGroup?.(spanGrouping[0].span, occurrence);
+        isEmbeddedSpanTree &&
+          trackAdvancedAnalyticsEvent(
+            'issue_details.performance.autogrouped_siblings_toggle',
+            {organization}
+          );
+      }}
```

Comment:
In general it's better to turn this kind of thing into a 'onToggle' or 'afterX' hook since otherwise you're letting issue details leak into the span tree.

Reply:
I'm gonna merge this in for now and focus on getting the other analytics out before EA, but I'll come back to this after